functional-tests.sh

```diff
@@ -696,17 +696,13 @@ function test_copy_object_with_sse()
     assert_success "$start_time" "${FUNCNAME[0]}" show_on_failure $? "unable to download object using 'mc cat'"
     assert_success "$start_time" "${FUNCNAME[0]}" check_md5sum "$FILE_1_MB_MD5SUM" "${object_name}.downloaded"
     assert_success "$start_time" "${FUNCNAME[0]}" mc_cmd rm "${object_name}.downloaded"
-    # mc rm on encrypted object without encryption key should fail
-    assert_failure "$start_time" "${FUNCNAME[0]}" mc_cmd rm "${SERVER_ALIAS}/${BUCKET_NAME}/${prefix}/${object_name}"
-    # mc rm on encrypted object with encryption key should pass
-    assert_success "$start_time" "${FUNCNAME[0]}" mc_cmd rm --encrypt-key "${cli_flag}" "${SERVER_ALIAS}/${BUCKET_NAME}/${prefix}/${object_name}"
-    # mc rm on unencrypted destination object should pass
-    assert_success "$start_time" "${FUNCNAME[0]}" mc_cmd rm  "${SERVER_ALIAS}/${BUCKET_NAME}/${object_name}"
```

Comment:
cleanup of destination object is still needed.

Reply:
what ??

Reply:
```     assert_success "$start_time" "${FUNCNAME[0]}" mc_cmd rm  "${SERVER_ALIAS}/${BUCKET_NAME}/${object_name}" ```

Reply:
the destination object is same as source object  here @poornas

Reply:
No, they are different Harsha. Please look at https://github.com/minio/mc/pull/2901/files#diff-71b55f1605a99785eb285feff46dc57eL693

Reply:
Fixed @kannappanr